include/fmt/format.h

```diff
@@ -4105,7 +4105,7 @@ struct udl_formatter {
 
 template <typename Char>
 struct udl_arg {
-  const Char *str;
+  basic_string_view<Char> str;
```

Comment:
Please leave `udl_arg` as before. We want to discourage the use of `\0` in argument names.

Reply:
@vitaut ok. Is this for performance reasons as you mentioned on CppCon 2017?

Reply:
I am not very concerned about performance here, but using a null character in argument names (as opposed to the format string itself) seems like a bad idea because these names should normally be readable.